### DISCUSSION TITLE
Documentation on `Includes`

### DISCUSSION THREAD

**Discussion by AdaemmerP** [quarto-dev/quarto-cli#11626] [Q&A]
### What would you like to do?

Report an issue on quarto.org

### Description

Thanks for the amazing Quarto!

This [section](https://quarto.org/docs/authoring/includes.html) on including and running external scripts is a bit confusing and leaves some questions unanswered.

For example, why does 

```` markdown
```python
{{< include _demo.py >}}
```
````

copy the Python code above and run it, whereas 

```` markdown
```r  
{{< include _demo.R  >}}
```
```` 

only copies the code, but does not run it? 

In addition, what are other options and best practices to include and run external (R) scripts?

I found 

```` markdown
```{r} 
source("_demo.R")
```
````

and 

```` markdown
```{r} 
#| file: path/to/_demo.R
```
````
Any advice or improvements to the documentation would be greatly appreciated.

**Comment by cscheid** [comment]
I think you're confused. The following does _not_ run Python:

````
---
title: hello
---

```python
{{< include _demo.py >}}
```
````

**Reply by AdaemmerP** [reply]
Thanks and true. I got confused with the sentence `Copy the Python code above and run it.` 
Still, it would be nice to hear some best practices in Quarto to include and run R scripts in `.qmd` files.

**Comment by cscheid** [comment]
I'm going to convert this to a Q&A.
